app/assets/stylesheets/marketing_redesign/components/_section.css

```
@@ -1,9 +1,10 @@
-section.section {
+:where(section.section) {
   align-items: center;
   display: grid;
   gap: var(--space--base);
   grid-template-columns: repeat(12, [col-start] 1fr);
   justify-content: center;
+  justify-content: center;
```

Looks like property is duplicated.

```suggestion
  justify-content: center;
```